.github/workflows/linux-ci.yml

```diff
@@ -52,4 +49,10 @@ jobs:
           -DPython_ADDITIONAL_VERSIONS=2.7 -DPLASMA_BUILD_TESTS=ON \
           -DPLASMA_BUILD_TOOLS=ON -DPLASMA_BUILD_RESOURCE_DAT=ON ..
         make -j2
+
+    - name: Test Plasma
+      run: |
+        cd ${GITHUB_WORKSPACE}/build
         make check
+      env:
+        CTEST_OUTPUT_ON_FAILURE: 1
```

Comment:
I wonder if this could be done as an argument to the ctest command?

Reply:
Apparently yes, as `--output-on-failure`.
It can also be set as a default argument to ctest in the CMakeLists:
```
list(APPEND CMAKE_CTEST_ARGUMENTS "--output-on-failure")
```

Do we ever *not* want to see the output of the failing test on failure? Should this be turned on by default in our main CMakeLists?

Reply:
I can't think of any reason why *not* seeing the test output would be desirable.